extensions/machine-learning/src/views/widgets/dashboardWidget.ts

```diff
@@ -405,37 +405,22 @@ export class DashboardWidget {
 			width: maxWidth + 10,
 			justifyContent: 'flex-start'
 		}).component();
-		const linkComponent = view.modelBuilder.hyperlink().withProperties({
-			label: linkMetaData.title,
-			url: linkMetaData.link,
+		const linkComponent = view.modelBuilder.hyperlink().withProps({
+			label: linkMetaData.title!,
```

Comment:
Why '!' needed now when it was not needed before? What changed?

Reply:
as I explained in the comment above, previously, you can pass in any value, withProps is strong typed, I have to pass in a string value.